packages/graphql-serve/README.md

```diff
@@ -60,14 +60,15 @@ type Note {
 
 Assuming you have created your various `*.graphql` data model files in the `models` directory, to automatically generate resolvers and start a GraphQL server listening on port 8080 do the following:
 
-For npm:
 ```
 $ gqlserve --model=models --port=8080
 ```
 
-For yarn:
+
+If you only need to see the generated GraphQL Schema, use the `--schemaOnly` option:
+
 ```
-$ yarn gqlserve --model=models --port=8080
+$ gqlserve --schemaOnly
```

Comment:
This should be command like print schema.

Reply:
Should we also make serve a command? This way it'll have two commands serve and print schema

Reply:
Ok

Reply:
Ok